### DISCUSSION TITLE
NextJS Starter Crashes When I Follow The Instructions.

### DISCUSSION THREAD

**Discussion by noraa-july-stoke** [markdoc/markdoc#433] [Bug report] [answered]
Was attempting to write documentation for a product API....
I am on an Apple Silicon macbook pro M2 using chrome browser if that is helpful. 

I simply follow the steps from the starter, and I get an error. I dug around in the code base to see what props were being passed down to "MyApp", but I couldn't find anything. 

To be clear, I took these exact steps.... As those are the only steps in the repo.
1. Cloned the repo.
2. `yarn install`
3. `yarn dev`

![Screenshot 2023-08-09 at 3 49 37 PM](https://github.com/markdoc/markdoc/assets/109265344/2b17a27a-7784-4f52-8a39-b3b6bfa7e191)

**Comment by mfix-stripe** [comment, marked as answer]
Sorry about that @noraa-july-stoke! If you reinstall, it should be fixed now.

**Comment by noraa-july-stoke** [comment]
@mfix-stripe that was fast, thanks!
